inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/BinlogReader.java

```diff
@@ -152,12 +162,14 @@ public void init(JobProfile jobConf) {
         finished = false;
 
         offset = jobConf.get(JOB_DATABASE_OFFSETS, "");
+        specificOffsetFile = jobConf.get(JOB_DATABASE_OFFSET_SPECIFIC_OFFSET_FILE, "");
```

Comment:
please add the parameter in inlong manager and inlong manager client

Reply:
next commit